libs/wingsdk/src/util/enhanced-error.ts

```diff
@@ -65,15 +66,27 @@ export async function prettyPrintError(
 
   const message = fBold(fRed("runtime error: ")) + fRed(originalMessage);
 
-  st = await st.clean().withSourcesAsync();
+  st = await st
+    .clean()
+    .filter((item) => !item.native)
+    // strip node internals
+    .filter((item) => !item.file.startsWith("node:"))
+    // strip wingsdk
+    .filter((item) => !normalPath(item.file).includes("/libs/wingsdk/src/"))
+    // special: remove the handler wrapper (See `cloud.Function` entrypoint for where this comes from)
+    .filter((item) => !normalPath(item.file).match(/\.wing\/handler_\w+\.js$/))
+    .withSourcesAsync();
```

Comment:
I wonder if there's any way to add some test cases for these... maybe we can add some kind of `stack_traces.test.w` to the error or invalid directory with examples of different kinds of stack traces (for cases where the root error comes from a winglib, or from Wing SDK, or from CDKTF, or from constructs, or from `assert` or `bring expect;`, or from the AWS SDK, etc.). Just random ideas :-)